lara-typescript/src/rubric-authoring/rubric-general-options.tsx

```diff
@@ -65,33 +66,29 @@ export const RubricGeneralOptions = () => {
             <td className="rubric-checkbox">
               <input
                 type="checkbox"
-                name="showRatingDescriptions"
-                checked={rubric.showRatingDescriptions}
-                onChange={handleUpdateCheckbox("showRatingDescriptions")}
-              /> Show rating descriptions
-            </td>
-          </tr>
-          <tr>
-            <td/>
-            <td className="rubric-checkbox">
-              <input
-                type="checkbox"
-                name="scoreUsingPoints"
-                checked={rubric.scoreUsingPoints}
-                onChange={handleUpdateCheckbox("scoreUsingPoints")}
-              /> Score using points
+                name="hideRubricFromStudentsInStudentReport"
+                checked={rubric.hideRubricFromStudentsInStudentReport}
+                onChange={handleUpdateCheckbox("hideRubricFromStudentsInStudentReport")}
```

Comment:
If you're always using the `name` prop, I sometimes have handlers where the key is obtained from `name`, like `const name = e.target.name`. But then you lose type checking, as you can't enforce it for `name`.